components/Table/Table.js

```diff
@@ -85,7 +96,63 @@ export class Table {
       menuBtn.classList.add('menuBtn');
       menuBtn.textContent = 'more_vert';
 
-      actionsContainer.append(arrowBtn, menuBtn);
+      const dropDownMenu = document.createElement('div');
+      dropDownMenu.classList.add('drop-down__menu');
+      const menuItems = this.menuConfig.map((item) => {
+        const menuItem = document.createElement('a');
+        menuItem.classList.add('menu-item');
+        menuItem.textContent = item.label;
+
+        if (!col.sortable) {
+          if (item.key === 'sortbyasc') {
```

Comment:
1. Please, fix the name of these keys. I'm talking about `sortbyasc` and etc. - it's hard to read. Use camelCase or at least underscore
2. Let's move these values as a constant out of the class. We could create the map(object) for this and use it here